src/data/finset/lattice.lean

```diff
@@ -452,6 +460,14 @@ lemma inf_eq_infi [complete_lattice β] (s : finset α) (f : α → β) : s.inf
 lemma inf_id_eq_Inf [complete_lattice α] (s : finset α) : s.inf id = Inf s :=
 @sup_id_eq_Sup (order_dual α) _ _
 
+lemma inf_id_eq_sInter {α} (s : finset (set α)) :
+  s.inf id = ⋂₀(↑s) :=
+inf_id_eq_Inf _
+
+@[simp] lemma inf_eq_bInter {α β : Type*} {s : finset α} {f : α → set β} :
+  s.inf f = ⋂ x ∈ s, f x :=
+inf_eq_infi _ _
```

Comment:
I assume there's already an `α` and `β` in scope. Also probably good to put `set` in the name
```suggestion
@[simp] lemma inf_id_set_eq_sInter (s : finset (set α)) : s.inf id = ⋂₀(↑s) :=
inf_id_eq_Inf _

@[simp] lemma inf_set_eq_bInter (s : finset α) (f : α → set β) : s.inf f = ⋂ x ∈ s, f x :=
inf_eq_infi _ _
```